src/onegov/town6/forms/chat.py

```diff
@@ -41,8 +41,8 @@ class ChatInitiationForm(Form):
     confirmation = BooleanField(
         label=_("Confirmation"),
         description=_(
-            "Ich bestätige, dass mir bewusst ist, dass dieser Chat "
-            "gespeichert und der Verlauf an mich per Mail gesendet wird."),
+            "I confirm that I am aware that this chat will be saved and the "
```

Comment:
![Screenshot from 2024-01-29 11-19-57](https://github.com/OneGov/onegov-cloud/assets/36520284/760b3a55-e125-4b7d-8d36-f5e9cdf7e251)

translation is showing english.